docs/endpoints/users/settings.md

```diff
@@ -1,3 +1,52 @@
 # /\_settings
 
+## Overview
+
+The `/_settings` endpoint allows users to update their profile settings and change their password. It is designed to offer users control over various aspects of their account settings through a single interface.
+
 ## POST
+
+The `POST` method is used to update user settings and change passwords.
+
+### Global Response Codes
+
+These are common HTTP status codes applicable to all requests made to this endpoint:
+
+- `200` OK: Returns the updated user object.
+- `400` Bad Request: The server cannot process the request due to a client error (e.g., malformed request syntax).
+- `401` Unauthorized: Authentication is required and has failed or has not been provided.
+- `429` Too Many Requests: The user has sent too many requests in a given amount of time ("rate limiting").
+- `500` Internal Server Error: The server has encountered a situation it doesn't know how to handle. If this error occurs, please create an issue on [GitHub](https://github.com/discuitnet/discuit).
+
+### Update Profile
+
+Allows users to update various settings related to their profile. For this request you must set `?action=updateProfile` in the URL.
+
+#### Request Body
+
+```json
+{
+  "aboutMe": "string",
+  "upvoteNotificationsOff": "boolean",
+  "replyNotificationsOff": "boolean",
+  "homeFeed": ["All", "Subscriptions"],
+  "rememberFeedSort": "boolean",
+  "embedsOff": "boolean",
+  "email": "string",
+  "hideUserProfilePictures": "boolean"
+}
```

Comment:
I think instead of using a JSON object, we should use a type definition. I've been using TypeScript-style definitions so far because they're fairly simple to understand in my opinion.

In this case we can use something like:

```ts
type Request = {
  aboutMe: string;
  upvoteNotificationsOff: boolean;
  replyNotificationsOff: boolean;
  homeFeed: "All" | "Subscriptions";
  rememberFeedSort: boolean;
  embedsOff: boolean;
  email: string;
  hideUserProfilePictures: boolean;
}
```

Reply:
The only change I'd make to the TypeScript definition style is changing the optional types to explicitly say undefined. I've used the `?` on other pages, and I need to change that to `| undefined`.